float32 angle
---
float32 cosValue
float32 sinValue
